lib/Test2/Event/TAP/Version.pm

```diff
@@ -0,0 +1,17 @@
+package Test2::Event::TAP::Version;
```

Comment:
I feel like this event, or specifically one similar belongs in Test2... but I am not really sure.